DeploymentCloud/Deployment.Common/deployResources.ps1

```diff
@@ -633,6 +636,15 @@ function Setup-Secrets {
 
     $secretName = $prefix + "datax-sa-" + $sparkBlobAccountName    
     Setup-Secret -VaultName $vaultName -SecretName $secretName -Value $tValue
+	
+	if($setupAdditionalResourcesForSample -eq 'y'){
+		$secretName = $prefix + "datax-sa-fullconnectionstring-" + $sparkBlobAccountName    
+		Setup-Secret -VaultName $vaultName -SecretName $secretName -Value $storageAccount.Context.ConnectionString
+	
+		$sqlCosmosdbCon = Get-CosmosDBConnectionString -Name $sqlDocDBName
+		$secretName = "output-samplecosmosdb"
```

Comment:
For consistency, can we have a prefix for this secret name? we usually have one for a secret name.

Reply:
moved to deploySample.ps1 and there is no prefix in the deploySample